gogigak/urls.py

```diff
@@ -1,10 +1,12 @@
 from django.urls     import path, include
 
 from products.views  import CategoryImageView
+from users.views     import DeliveryView
 
 urlpatterns = [
     path('users', include('users.urls')),
     path('orders', include('orders.urls')),
     path('products', include('products.urls')),
     path('categories', CategoryImageView.as_view()),
+    path('address', DeliveryView.as_view()),
```

Comment:
```suggestion
    path('delivery/check', DeliveryView.as_view()),
```
check가 명사형도 있으니까 괜찮을 것 같아요

Reply:
수정했습니다!